README.md

```diff
@@ -136,6 +135,38 @@ span-fabric poseidon
 
 BCF is now configured and ready for use with Poseidon.
 
+## Configuring Poseidon
+
+You will need to create a directory and config file on the server where Poseidon will run.
+
+```
+sudo mkdir /opt/poseidon
+sudo cp config/poseidon.config /opt/poseidon
+```
+
+Now, edit this file. You will need to set at minimum:
+
+* controller_type, as appropriate to the controller you are running (see above).
+* collector_nic: must be set to the interface name on the server, that is connected to the switch mirror port.
+* controller_mirror_ports: must be set to the interface on the switch that will be used as the mirror port.
+
+
+## Starting Poseidon
```

Comment:
This whole section should be part of the `Usage` section that runs through using the `poseidon` command to install/start, etc. There should then be an update to `bin/poseidon` for these docker-compose updates.